python-avd/pyavd/_eos_designs/schema/schema_fragments/wan_use_sso_for_route_injection.schema.yml

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2024 Arista Networks, Inc.
+# Use of this source code is governed by the Apache License 2.0
+# that can be found in the LICENSE file.
+# yaml-language-server: $schema=../../../_schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  wan_use_soo_for_route_injection:
+    documentation_options:
+      table: wan-settings
+    description: |-
+      In AVD 5.0.0, tags are used instead of SOO to manage which routes are
```

Comment:
```suggestion
      By default, route tags are used instead to mark which routes are exported to EVPN.
```

Reply:
We should not talk about versions in schemas, as they will quickly become outdated. It would be better to talk about the two behaviors.
The version stuff needs to go in the porting guide.